packages/teleport/src/services/auth/auth.ts

```diff
@@ -19,10 +19,15 @@ import 'u2f-api-polyfill';
 import api from 'teleport/services/api';
 import cfg from 'teleport/config';
 import makePasswordToken from './makePasswordToken';
+import {
+  makeMfaAuthenticateChallenge,
+  makeMfaRegistrationChallenge,
+} from './makeMfa';
+import { DeviceType } from './types';
 
 const auth = {
   u2fBrowserSupported() {
-    if (window.u2f) {
+    if (window['u2f']) {
```

Comment:
You can also do a

```typescript
declare global {
  interface Window {
    u2f: any;
  }
}
```
to shut up typescript. Just an fyi in case you'd never seen it.